src/WordPress/Blueprints/Resources/Resolver/FilesystemResourceResolver.php

```diff
@@ -24,12 +24,12 @@ public function supports( ResourceDefinitionInterface $resource ): bool {
 		return $resource instanceof FilesystemResource;
 	}
 
-	public function stream( ResourceDefinitionInterface $resource, Tracker $progressTracker ) {
+	public function stream( ResourceDefinitionInterface $resource, Tracker $progress_tracker ) {
```

Comment:
Is this coming from WordPress coding standards?

Reply:
Yeah.

Reply:
Ugh. Alright.